spec/features/user_can_signup.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
```

Comment:
Everything passing ?

Reply:
Yep, apart from the user can submit post feature test of yours which is waiting to be merged with what Yousuf & Suhani wrote which resets the indexing before each spec